autopush/tests/test_endpoint.py

```diff
@@ -173,7 +170,7 @@ def test_update_too_much_data(self):
 
         def handle_finish(result):
             self.assertTrue(result)
-            self.status_mock.assert_called_with(401)
+            self.status_mock.assert_called_with(413)
```

Comment:
I think there could be some better checks though, I'll add a few more to ensure the error is formatted right, all the fields are present, etc.